internal/services/subscription-service/docker-compose.yaml

```
@@ -32,12 +32,49 @@ services:
+    command:
+      - '--promscrape.config=/etc/prometheus/prometheus.yml'
+      - '--promscrape.config.strictParse=false'
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml:ro
+    environment:
+      - GF_SECURITY_ADMIN_PASSWORD=admin
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Viewer
+    ports:
+      - "${VICTORIA_METRICS_PORT:-8428}:8428"
+    networks: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)